src/train.py

```diff
@@ -0,0 +1,23 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL']='3'
+import tensorflow as tf
+import argparse
+import subprocess
+import os
+import importlib
+from util import load_hparams, parse_train_args
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Start training a model')
+
+    parser.add_argument('--model', metavar='N', required=True, help='The model to train', type=str, nargs='?')
```

Comment:
Are you sure this should be `nargs='?'`? Also, the `metavar='N'` probably shouldn't be there; it only affects the --help message, and `N` is usually only for integers.

Also, might be worth looking into [subcommands](https://docs.python.org/3/library/argparse.html#sub-commands) rather than parsing twice; among other things, it would make help messages more useful.

Reply:
Yeah, I just repurposed the code @christoffer-arvidsson wrote. I haven't used argparse before.

Reply:
It made sense to use `nargs='?'` when you are fine with choosing default if no argument is supplied.

Reply:
Create issue